fixes #1725 (#1726)
nf-core/nf-co.re: markdown/developers/adding_pipelines.md

then your life will be much easier.
This tool does lots of things for you: it gives you the correct file structure and boiler plate code
and also sets up the required `git` infrastructure for you to keep your pipeline in sync in the future.

> Be sure to set the default branch to `dev` before start working on the code, by moving inside the newly 

Reviewer: ```suggestion
Be sure to switch to the `dev` branch before start working on the code, by moving inside the newly 
```

created pipeline folder and using the command:

> ```
> git checkout dev
> ```

Reviewer: ```suggestion
    ```bash
    git checkout dev
    ```
    
    or 
    
    ```bash
    git switch dev
    ```
```

Even if you already have a working pipeline, it may be easier in the long run to use this this template
and copy over your code in the relevant places.

> Once the template for your pipeline is created, make sure to switch branch to the `dev` branch with `git switch dev`.

Reviewer 2: I think let's stick to the more standard / well known git commands for now..

```suggestion
Once the template for your pipeline is created, make sure to switch branch to the `dev` branch with `git checkout dev`.
```
